src/services/formly.config.ts

```diff
@@ -1,11 +1,11 @@
-import {Injectable, ViewContainerRef, Type} from "@angular/core";
+import {Injectable, ViewContainerRef} from "@angular/core";
 
 /**
  * Maintains list of formly field directive types. This can be used to register new field templates.
  */
 @Injectable()
 export class FormlyConfig {
-  types: {[name: string]: Type} = {};
+  types: {[name: string]: any} = {};
   manipulators: {[name: string]: TemplateManipulator} = {};
```

Comment:
Should we stick to strict typing instead of using any. We get better compile time errors with strict typing instead of getting runtime errors.

May not be big deal but good to have.